docs/docs/40-contributor-guide/15-release-procedures.md

```diff
@@ -154,6 +154,29 @@ release date -- which is always a Friday.
       [automated release process](https://github.com/akuity/kargo/actions/workflows/release.yaml)
       to complete.
 
+1. Update [Homebrew Formula](https://github.com/akuity/homebrew-tap).
+
+    * Clone the Homebrew tap repository:
+        ```bash
+        git clone https://github.com/akuity/homebrew-tap.git
+        cd homebrew-tap
+        ```
+    * Run the update script with the new version:
+        ```bash
+        ./update.sh kargo vM.m.0
+        ```
+    * Verify the updated formula:
+        ```bash
+        brew install --build-from-source ./kargo.rb
+        kargo --version
+        ```
+    * Commit and push the changes to the Homebrew tap repository:
+        ```bash
+        git add kargo.rb
+        git commit -m "Update Kargo formula to version vM.m.0"
+        git push origin main
+        ```
```

Comment:
I'd rather we automated this. It should just be a new job in the release process.

Reply:
It may at this point be easier to start looking into something like [GoReleaser](https://goreleaser.com), which has built-in utilities to update Brew formulas (in different repositories: [example](https://github.com/fluxcd/flux2/blob/f42a17de541d6ce2ffe56d0dcba96a26ce581818/.goreleaser.yml#L74-L88)).

Reply:
She had been looking at that. I meant to read up as well.

Reply:
> I will take a look and see how GitHub Actions can run scripts in a different repository

goreleaser, aside, this isn't hard. The a job in one repo can easily clone a different repo, make changes (by running a script, if that's the best way), and commit. Preferable to committing directly would be opening a PR because that way we can still have branch protections on the tap repo's main branch.